mergedb - UnboundLocalError: local variable 'bits' referenced before assignment

```
make[2]: Leaving directory '/source/fuzzers/010-clb-lutinit'
[91m+ test 2 = 2
+ test -e build/segbits_clbll.db
++ mktemp -p .
+ tmp1=./tmp.fiy129s2D6
++ mktemp -p .
+ tmp2=./tmp.DfRbqH96vb
+ trap finish EXIT
+ db=/source/database/artix7/segbits_clbll_l.db
+ '[' -f /source/database/artix7/segbits_clbll_l.db ']'
+ python3 /source/utils/parsedb.py --strict build/segbits_clbll.db
+ ismask=false
+ case "$1" in
+ sed -e 's/^CLB\.SLICE_X0\./CLBLL_L.SLICEL_X0./' -e 's/^CLB\.SLICE_X1\./CLBLL_L.SLICEL_X1./'
+ touch /source/database/artix7/segbits_clbll_l.db
+ false
+ python3 /source/utils/mergedb.py --out ./tmp.DfRbqH96vb /source/database/artix7/segbits_clbll_l.db ./tmp.fiy129s2D6
Traceback (most recent call last):
  File "/source/utils/mergedb.py", line 64, in <module>
    main()
  File "/source/utils/mergedb.py", line 60, in main
    verbose=args.verbose)
  File "/source/utils/mergedb.py", line 37, in run
    entries[tag] = bits
UnboundLocalError: local variable 'bits' referenced before assignment
+ finish
+ echo 'Cleaning up temp files'
+ rm -f ./tmp.fiy129s2D6
+ rm -f ./tmp.DfRbqH96vb
make[2]: *** [pushdb] Error 1
[0mMakefile:24: recipe for target 'run' failed
[91mmake[1]: *** [run] Error 2
[0m[91mmake: *** [010-clb-lutinit/run.ok] Error 2
[0mMakefile:27: recipe for target '010-clb-lutinit/run.ok' failed
The command '/bin/sh -c bash -c ". /source/database/artix7/settings.sh; cd /source/fuzzers && make --output-sync=target --warn-undefined-variables"' returned a non-zero code: 2
ERROR
```

--- a/utils/mergedb.py
+++ b/utils/mergedb.py
@@ -34,10 +34,10 @@ def run(fn_ins, fn_out, strict=False, verbose=False):
                     print("  New line : %s" % line)
                     assert not strict, "strict: got duplicate bits"
 
-        entries[tag] = bits
-        tags[tag] = (bits, line)
-        if bits != None:
-            bitss[bits] = (tag, line)
+            entries[tag] = bits
+            tags[tag] = (bits, line)
+            if bits != None:
+                bitss[bits] = (tag, line)
 
     util.write_db_lines(fn_out, entries)
 

